app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/logging_mixin.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+# Encapsulate logging methods into a single mixin.
+module ManageIQ::Providers::IbmCloud::VPC::CloudManager::LoggingMixin
+  # A logger wrapper that can be used as a general interface during dev or production.
+  # During development use the defined $ibm_cloud_log which prints to ibm_cloud.log.
+  # In production use the defined _log which prints to general evm log and should be captured in containers.
```

Comment:
Anything logged to `$ibm_cloud_log` will be printed to stdout in containers since it is a multicast logger